packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -465,15 +469,25 @@ export const Async: ComponentStory<typeof FilterMultiSelect> = args => {
     [data]
   )
 
+  // Combine current and cached and remove the duplicates
```

Comment:
Include a description of the feature that this code is supporting

Reply:
updated :)